#! /bin/sh /usr/share/dpatch/dpatch-run
## 01-man.dpatch by Stuart Brady <sdbrady@ntlworld.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Refer to fuse-emulator instead of fuse in the man page, due to the
## DP: altered binary name (so as not to conflict with the
## DP: Filesystem in USErspace packages.)

@DPATCH@
--- fuse-emulator-0.9.0.orig/man/fuse.1
+++ fuse-emulator-0.9.0/man/fuse.1
@@ -1,6 +1,6 @@
 .\" -*- nroff -*-
 .\"
-.\" fuse.1: Fuse man page
+.\" fuse-emulator.1: Fuse man page
 .\" Copyright (c) 2001-2007 Russell Marks, Philip Kendall, Darren Salt,
 .\"                         Fredrick Meunier, Stuart Brady
 .\"
@@ -23,12 +23,12 @@
 .\" E-mail: philip-fuse@shadowmagic.org.uk
 .\"
 .\"
-.TH fuse 1 "5th January, 2008" "Version 0.9.0" "Emulators"
+.TH fuse-emulator 1 "5th January, 2008" "Version 0.9.0" "Emulators"
 .\"
 .\"------------------------------------------------------------------
 .\"
 .SH NAME
-fuse \- Sinclair ZX Spectrum emulator
+fuse\-emulator \- Sinclair ZX Spectrum emulator
 .\"
 .\"------------------------------------------------------------------
 .\"
@@ -36,7 +36,7 @@
 .\" the trend for long-option-using programs is to give a largely
 .\" generic synopsis, so...
 .PD 0
-.B fuse
+.B fuse\-emulator
 .RI [ options ]
 .P
 .PD 1
@@ -835,11 +835,11 @@
 .PP
 Some examples of use:
 .PP
-.I fuse \-d alsa09:dev=hw:1
+.I fuse\-emulator \-d alsa09:dev=hw:1
 .PP
 causes Fuse to use ALSA 0.9+ output with the second (#1) sound card.
 .PP
-.I fuse \-d raw:byteorder=little,file=enigma.raw
+.I fuse\-emulator \-d raw:byteorder=little,file=enigma.raw
 .PP
 causes Fuse to save little endian words to
 .RI ` enigma.raw "'."
@@ -898,13 +898,13 @@
 .PP
 Some examples of use:
 .PP
-.I fuse \-d verbose,buffer=2000
+.I fuse\-emulator \-d verbose,buffer=2000
 .PP
 causes Fuse to use the default ALSA device with 2000 frame length buffer and
 report ALSA buffer underruns on
 .IR stderr .
 .PP
-.I fuse \-d tee:plughw:0,\(aq/tmp/aufwm.raw\(aq,raw
+.I fuse\-emulator \-d tee:plughw:0,\(aq/tmp/aufwm.raw\(aq,raw
 .PP
 causes Fuse to use the first card and parallel save the raw audio samples
 into
@@ -3287,9 +3287,9 @@
 other than SDL.
 .PP
 Attempting to replay an RZX file without an embedded snapshot from the
-command-line doesn't work if you do `./fuse
+command-line doesn't work if you do `./fuse\-emulator
 .IR "rzxfile snapshot" '.
-`./fuse
+`./fuse\-emulator
 .IR "snapshot rzxfile" '
 does work though, so use that instead.
 .PP
@@ -3322,7 +3322,7 @@
 .\"
 .SH SEE ALSO
 .IR bzip2 "(3),"
-.IR fuse\-utils "(1),"
+.IR fuse\-emulator\-utils "(1),"
 .IR gzip "(3),"
 .IR libspectrum "(3),"
 .IR ogg123 "(1),"
